server/src/LspServer.ts

```diff
@@ -187,21 +188,32 @@ export class LspServer {
   }
 
   async prepareDestination(profileResult: Result<DbtProfileSuccess, DbtProfileError>): Promise<void> {
-    if (profileResult.isOk()) {
-      const bigQueryContextInfo = await BigQueryContext.createContext(profileResult.value);
+    if (
+      profileResult.isOk() &&
+      profileResult.value.dbtProfile &&
```

Comment:
Looks like if `profileResult.value.dbtProfile !== undefined` it contains BigQuery profile, so may be we can avoid following checks?